lib/loader/loader.c

```diff
@@ -42,20 +42,14 @@
 #define ATTR_FORMAT(...)
 #endif
 
-#if _WIN32
```

Comment:
Why is this change mixed in here?

Reply:
The `strcpy_s` from MSVC does not work in this new case. Instead of returning non-zero value it triggers some global handler to handle the security issue.

Reply:
I moved it as a first commit. We can land it separately, but I'd rather save out time.

Reply:
Its fine, the commit describes the reason now.